### PR TITLE
RHMAP-3447 bump fh-db version

### PR DESCRIPTION
**Pull request by pb82**
bump fh-db version

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by pb82** [comment]
retest this please

**Comment by pb82** [comment]
retest this please

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.9",
+  "version": "5.13.11",
```
Comment (matzew) [inline package.json]: @pb82 what's the reason for jumping from .9 -> 11 ? 

Also, why was there no tag created as part of the release/push to npmjs ?

Comment (pb82) [inline package.json]: @matzew I released 5.13.10 as well but then made another change to fh-db and had to bump the version again. Sorry, forgot to tag it. will do.

Comment (matzew) [inline package.json]: I think generally a huge problem is that we run releases for every change

**Pull request merged**